toybox/src/lib.rs

```diff
@@ -16,10 +16,10 @@ pub fn get_simulation_by_name(name: &str) -> Result<Box<dyn Simulation>, String>
         "breakout" => Ok(Box::new(breakout::Breakout::default())),
         #[cfg(feature = "gridworld")]
         "gridworld" => Ok(Box::new(gridworld::GridWorld::default())),
-        #[cfg(feature = "space_invaders")]
```

Comment:
I'd be OK if we ditched the cfg-feature per game situation. We'd initially envisioned compiling just the games you need, but the rust compiler has gotten faster and we're distributing on pip now...